nodedb.go

```diff
@@ -347,26 +347,28 @@ func (ndb *nodeDB) deleteOrphansHelper(db dbm.DB, batch dbm.Batch, flushToDisk b
 	}
 }
 
-func (ndb *nodeDB) PruneRecentVersions() (prunedVersions []int64, err error) {
+func (ndb *nodeDB) PruneRecentVersion() (int64, error) {
```

Comment:
Note we only return a single value. We never returned more than one so I'm not sure why we originally returned an array...

Reply:
I'm just guessing, but he idea was perhaps to return which versions were pruned, but it seems like it was changed to only prune one version at a time.